addons/test_mail/tests/test_mail_gateway.py

```diff
@@ -349,7 +349,6 @@ def test_message_process_followers(self):
                 'email': 'private.customer@text.example.com',
                 'phone': '0032455112233',
                 'name': 'Private Customer',
-                'type': 'private',
```

Comment:
I think this makes the ``partner_private`` variable not private, and not useful anymore :disguised_face: